src/google-mobility.html

```diff
@@ -11,23 +11,34 @@
       <div class="rds-hero" id="home">
         <div class="container mdc-layout-grid">
           <div class="header">
+            <a href="https://www.google.com/covid19/mobility/"><img class="google-logo" src="./assets/Google__G__Logo.svg" /></a>
             <h1 class="mdc-typography--headline1">
               See how community movement is impacted by COVID-19.
             </h1>
+            <a class="rds-button-secondary" href="./covid.html">
+              <i class="material-icons mdc-button__icon" aria-hidden="true">open_in_new</i>RDS COVID-19 Project</a
+            >
             <a class="rds-button-flat-white" href="https://www.richdataservices.com" target="_blank">
               <i class="material-icons mdc-button__icon" aria-hidden="true">open_in_new</i>What is RDS?
             </a>
+
+            <a class="rds-button-flat-white" href="https://www.google.com/covid19/mobility/" target="_blank">
+              <i class="material-icons mdc-button__icon" aria-hidden="true">open_in_new</i>About Google Community Reports
+            </a>
+            <p class="banner-footnote mdc-typography--caption">
```

Comment:
On large screen, the text displays next to the button. 
<img width="1711" alt="Screen Shot 2020-07-18 at 1 14 17 PM" src="https://user-images.githubusercontent.com/11969737/87857945-9d04bf00-c8f8-11ea-9859-509f7a0d3ce8.png">